frontend/src/components/Login.svelte

```diff
@@ -0,0 +1,62 @@
+            style="fill:rgb(44,44,44);fill-rule:nonzero;"></path>
+        </g>
+      </g>
+    </svg>
```

Comment:
just a thought for later, but might be a bit cleaner to put svgs in assets. think you can import them like a svelte component!

Reply:
Ref Brage's comment https://github.com/dotkom/vengeful-vineyard/pull/168#issuecomment-1299249195, seems to be fixed by adding the SVG directly in the file:o. Should probably be another cleaner solution out there though.